addon/styles/_variables.scss

```diff
@@ -1,9 +1,24 @@
 //default properties
+$brown: #a52a2a;
```

Comment:
I'm not sure why these variables are included here. Are they needed for something? I don't see that in the rest of the code.